### DISCUSSION TITLE
[Enhancement] - Flag must be accepted anywhere in the command.

### DISCUSSION THREAD

**Discussion by yahu1031** [serverpod/serverpod#504] [Q&A] [answered]
`serverpod create dummypod -f` was not accepted now. But ideally, the force flag must be accepted anywhere in the command. Such that the action/command(whole) of what we are doing is meant to be forced.

**Comment by vlidholt** [comment, marked as answer]
`serverpod create -f dummypod` ;)

**Reply by yahu1031** [reply]
haha, I know this does it. But ideally, the enha-req is to able to pass `-f` after the proj name.
